scripts/just.config.ts

```diff
@@ -32,6 +34,24 @@ export const build = () => {
       bundle: true,
       outExtension: { ".js": ".mjs" },
       plugins: [
+        {
+          name: "inline-assert",
+          setup(build) {
+            // TODO: switch to SWC or tree-sitter
+            build.onLoad({ filter: /apollo-forest-run/ }, async (args) => {
```

Comment:
question: why we filter it? Would be nice to have one building config for all. Also why not to also grep **devAssert** from `graphql-js`?

Reply:
1. Babel transforms are expensive at build-time. Without the filter it will be applied to every package and we don't want that. 
2. I need this for ForestRun, but other packages should be opted-in as needed (and patterns adjusted accordingly). So far I don't know any other packages in our repo, needing this.